### PR TITLE
[MB-2099] Generate documentation & update build process for PCL.

### PR DESCRIPTION
**Pull request by madelgi**
[MB-2099] Generate documentation for PCL.

### CODE REVIEW

File: docs/build.gradle [hunk 1]
```diff
@@ -0,0 +1,16 @@
+task clean(type: Delete) {
```
Comment (rlepinski) [inline docs/build.gradle]: Do we want to use gradle here? can we do it with just an .sh file?

**Comment by rlepinski** [comment]
👍 

I am cool with using gradle but it would be nice to not have to use bash and gradle

File: docs/build.gradle (hunk 1 above)
Comment (rlepinski) [inline docs/build.gradle]: We should probably remove the gradle file for now

File: docs/mainpage.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+
+## Requirements
+ - Xamarin.iOS and Xamarin.Android
+
+## Resources:
+ - [Getting started guide](http://docs.urbanairship.com/platform/xamarin.html)
+
+## Setup
+Use the Xamarin component store or NuGeT to install the urbanairship-portable
```
Comment (hcrowell) [inline docs/mainpage.md]: NuGeT -> NuGet* ?

**Comment by hcrowell** [comment]
🍊  after reviewing comment above ^

**Pull request merged**